### PR TITLE
Resolved 'Call can throw, but it is not marked with 'try' and the error is not handled'

### PR DESCRIPTION
**Pull request by waseefakhtar**

### CODE REVIEW

**Comment by waseefakhtar** [comment]
I signed it!

File: ios/Swift/imagepicker/ImagePickerViewController.swift [hunk 1]
```diff
@@ -64,78 +64,84 @@ extension ViewController {
...
-            
-            // Check for errors
-            if (errorObj.dictionaryValue != [:]) {
-                self.labelResults.text = "Error code \(errorObj["code"]): \(errorObj["message"])"
-            } else {
-                // Parse the response
-                print(json)
-                let responses: JSON = json["responses"][0]
+            do {
+                let json = try JSON(data: dataToParse)
```
Comment (gguuss) [inline ios/Swift/imagepicker/ImagePickerViewController.swift]: This try is unneeded. You can just run as:

    let json =  JSON(data: dataToParse)

File: ios/Swift/imagepicker/ImagePickerViewController.swift [hunk 2]
```diff
@@ -64,78 +64,84 @@ extension ViewController {
...
+                        self.labelResults.text = labelResultsText
+                    } else {
+                        self.labelResults.text = "No labels found"
                     }
-                    self.labelResults.text = labelResultsText
-                } else {
-                    self.labelResults.text = "No labels found"
                 }
+                
+            } catch {
```
Comment (gguuss) [inline ios/Swift/imagepicker/ImagePickerViewController.swift]: This catch is unnecessary, just:

    }

is fine as there are no exceptions thrown in the code block.

File: ios/Swift/imagepicker/ImagePickerViewController.swift [hunk 3]
```diff
@@ -64,78 +64,84 @@ extension ViewController {
...
+                        
+                        self.faceResults.text = "People detected: \(numPeopleDetected)\n\nEmotions detected:\n"
                         
-                        // Sum all the detected emotions
-                        for emotion in emotions {
-                            let lookup = emotion + "Likelihood"
-                            let result:String = personData[lookup].stringValue
-                            emotionTotals[emotion]! += emotionLikelihoods[result]!
+                        var emotionTotals: [String: Double] = ["sorrow": 0, "joy": 0, "surprise": 0, "anger": 0]
+                        var emotionLikelihoods: [String: Double] = ["VERY_LIKELY": 0.9, "LIKELY": 0.75, "POSSIBLE": 0.5, "UNLIKELY":0.25, "VERY_UNLIKELY": 0.0]
```
Comment (gguuss) [inline ios/Swift/imagepicker/ImagePickerViewController.swift]: This is a long line continuation, maybe break up as:

                            var emotionLikelihoods: [String: Double] = [
                                "VERY_LIKELY": 0.9, "LIKELY": 0.75,"POSSIBLE": 0.5, "UNLIKELY":0.25, "VERY_UNLIKELY": 0.0]

Which just fits under 120 cols

File: ios/Swift/imagepicker/ImagePickerViewController.swift (hunk 1 above)
Comment (gguuss) [inline ios/Swift/imagepicker/ImagePickerViewController.swift]: Just to be clear, it sounds like this is why you submitted the PR: is JSON(...) throwing?

File: ios/Swift/imagepicker/ImagePickerViewController.swift (hunk 2 above)
Comment (gguuss) [inline ios/Swift/imagepicker/ImagePickerViewController.swift]: It sounds like this was why you submitted the PR, are there steps to reproduce the issue?

**Comment by AlcalaOrtegaRaulEnrique** [comment]
Hello, when I try build the example, I get: 
Gradle sync failed: Could not find method buildConfigField() android studio

**Pull request closed without merging**